AppInspector.CLI/Writers/AnalyzeHtmlWriter.cs

```diff
@@ -107,6 +114,39 @@ private void WriteJsonResult()
             WriteOnce.Verbosity = saveVerbosity;
         }
 
+
+
+        private string MergeResourceFiles(string inputPath)
+        {
+            StringBuilder stringBuilder = new StringBuilder();
+
+            if (Directory.Exists(inputPath))
+            {
+                try
+                {
+                    IEnumerable<string> srcfileList = Directory.EnumerateFiles(inputPath, "*.*", SearchOption.AllDirectories);
+                    if (srcfileList.Count() == 0)
```

Comment:
```suggestion
                    if (!srcfileList.Any())
```

Reply:
Cool optimization.  This function is removed for now and will be added back (more separation of goals) but I'll use this approach.